controllers/help-requests/help_requests.controller.js

```diff
@@ -175,6 +175,14 @@ module.exports = {
 
                 await HelpRequestsService.createHelpRequest(query, userName)
                 .then(result => {
+                    if(result.isError === true){
+                        let errorMessage = result.data || "Server error, please try later."
```

Comment:
I wouldn't show users of this service the raw error message - it'll probably be something technical and scary.

I suggest:
```suggestion
                        let errorMessage = "There's a problem with our service. Call our helpline at xxx-xxx-xxxx."
```

The tradeoff is we "lose" what these errors were, and they're harder to debug, but I think it's more important that a user (especially one in need of help!) doesn't get stuck.

Reply:
I don't know a phone number to put in there, maybe "Sorry, there is a problem with the service. Try again later" https://design-system.service.gov.uk/patterns/problem-with-the-service-pages/

Reply:
We could ask the team about this? 

Alternately, I think your suggestion is a good one - wouldn't argue with just using that for now, to get this done.